src/Raven.Server/Raven.Server.csproj

```diff
@@ -155,7 +155,12 @@
     <PackageReference Include="AWSSDK.Core" Version="3.7.302.16" />
     <PackageReference Include="AWSSDK.Glacier" Version="3.7.300.55" />
     <PackageReference Include="AWSSDK.S3" Version="3.7.305.31" />
+    <PackageReference Include="AWSSDK.Core" Version="3.7.302.9" />
+    <PackageReference Include="AWSSDK.Glacier" Version="3.7.300.49" />
+    <PackageReference Include="AWSSDK.S3" Version="3.7.305.25" />
```

Comment:
It seems like a merge conflict - see duplicated AWSSDK references

Reply:
done